api/auth.go

```diff
@@ -65,7 +66,20 @@ func (auth *Auth) CreateCustomerJWT(customerId int) ([]byte, error) {
 	return jwt.Sign(t, jwa.ES256, auth.sessionKey)
 }
 
-func (auth Auth) GetCustomerIDFromHeader(ctx echo.Context) (int, error) {
+func (auth *Auth) GetSessions() map[string]Session {
```

Comment:
note that `Session.Credential` might not get copied, because it's an `interface{}`

Reply:
Indeed. I'm not sure if there is a generic way to (deep?) copy any data. This won't be an issue at runtime as we only use this to get a base64 encoded verifiable presentation which is a string.